packages/medusa-payment-stripe/src/api/stripe/hooks/route.ts

```diff
@@ -1,11 +1,10 @@
 import { MedusaRequest, MedusaResponse } from "@medusajs/medusa"
-import { constructWebhook } from "../../utils/utils"
-
-const WEBHOOK_DELAY = process.env.STRIPE_WEBHOOK_DELAY ?? 5000 // 5s
-const WEBHOOK_RETRIES = process.env.STRIPE_WEBHOOK_RETRIES ?? 3
+import { constructWebhook, getPluginOptions } from "../../utils/utils"
 
 export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
   try {
+    const pluginOptions = getPluginOptions(req.scope)
```

Comment:
thought: Instead of creating this util, could we pull the options from the service here?

Something like:
```
const stripeService = req.scope.resolve("stripeProviderService")
const options = stripeService.options
```

Reply:
Good point! I had to add an accessor for options too in the service